.github/labeler.yml

```diff
@@ -0,0 +1,60 @@
+# Labels to be automatically added to PRs
+
+# Add 'config-change' label to any change to default config files
+config-change:
+  - "config.yaml.defaults"
+  - "test_config.yaml"
+
+# Add 'demo-data' label to any change to provided demo data
+demo-data:
+  - data/**/*
+
+# Add 'dependencies' label to any change to dependencies
+dependencies:
+  - "requirements.txt"
+  - "requirements.docs.txt"
+  - "package.json"
+
+# Add 'docker' label to any change to files related to Docker deployment
+docker:
+  - "Dockerfile"
+  - "docker.yaml"
+  - "docker-compose.yaml"
+  - ".dockerignore"
+
+# Add 'documentation' label to any change to the doc directory
+documentation:
+  - doc/**/*
+
+# Add 'formatting-and-linting' label to any changes to the formatting and
+# linting checks, and the pre-commit hooks that run them
+formatting-and-linting:
+  - ".eslintignore"
+  - ".eslintrc.yaml"
+  - ".flake8"
+  - ".git-pre-commit"
+  - ".pep8speaks.yml"
+  - ".pre-commit-config.yaml"
+
+# Add 'javascript' label to any change to *.js files in the static folder
+javascript:
+  - "static/js/**/*.js"
+  - "static/js/**/*.jsx"
+
+# Add 'migration' label to any change within the 'alembic' directory
+migration:
+  - alembic/versions/*
+
+# Add 'needs-migration?' label to any change to models.py without changes to
+# the Alembic versions
+needs-migration?:
+  - any: ["skyportal/models.py"]
```

Comment:
any way to see if the pinned baselayer/app/models.py was touched, too?

Reply:
Ah yeah good point - I'll look into it

Reply:
@acrellin Hmm I don't think this is straightforward to do. I enabled the debug logs for this action, and the only thing it gets back from the github API for files changed is `baselayer` and there is no info of what changed in the submodule.